### PR TITLE
adding folder input icon from Lucide

### PR DESCRIPTION
**Pull request by katefarrar**
Adding the `folderInput` icon from [Lucide](https://lucide.dev/icon/folder-input). Will be using this in the `Import` button in Import Services views.

grey example:
Screen Shot 2022-08-30 at 7 02 44 PM

red example:
Screen Shot 2022-08-30 at 7 02 54 PM

### CODE REVIEW

**Review by andreamorales — APPROVED** [review]

File: lib/components/icons/FolderInput.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Icon } from '@chakra-ui/react';
+import PropTypes from 'prop-types';
+import { colorOptions } from '../../theme/colors.theme';
+
+export const FolderInput = (props) => (
+  <Icon boxSize="24px" viewBox="0 0 24 24" {...props}>
+    <path
```
Comment (carissa-zukowski) [inline lib/components/icons/FolderInput.jsx:8]: @katefarrar from convo in slack (#team-design-system), if we want to move towards using the library, what is our motivation for pasting in the svg path here? if this icon is urgent we can coordinate with the FE team to prioritize. want to make sure that teams are aligned on how we want the DS to evolve and how FE is expected to interact.

Comment (katefarrar) [reply lib/components/icons/FolderInput.jsx:8]: hi @carissa-zukowski! I worked with @andreamorales & @leskat47 on this. the thinking was that we didn't want the Import Services work to be blocked by not having this icon. Happy to set up a time to discuss more or figure out another solution if this isn't optimal. Thanks!

**Review by theJakub — CHANGES_REQUESTED** [review]
We are trying to move away from this pattern. Until we have Lucide in our project and DS reworked, please pick an icon from react-icons for your designs. react-icons is already in laddertruck and ready to use.

**Comment by katefarrar** [comment]
Closing this without merging because we're going to use `RiFolderTransferLine` from [React Icons](https://react-icons.github.io/react-icons/). Thanks to everyone for the feedback and help!

**Pull request closed without merging**